django/template/defaultfilters.py

```diff
@@ -444,6 +444,16 @@ def escape_filter(value):
     return conditional_escape(value)
 
 
+@register.filter(is_safe=True)
+def escapeseq(value):
+    """
+    An "escape" filter for sequences. Mark each element in the sequence,
+    individually, as a string that should be auto-escaped. Return a list
+    with the results.
+    """
+    return [conditional_escape(obj) for obj in value]
```

Comment:
We could almost make this `[escape_filter(obj) for obj in value]`

Reply:
Also could've used the builtin `map()` here 😛 I ❤️ comprehensions but `map()` is good for applying another function… 
```
    return map(conditional_escape, value)
```